main.tf

```diff
@@ -35,16 +35,18 @@ resource "kubectl_manifest" "crds" {
   yaml_body = file("${path.module}/yamls/crds.yaml")
 }
 
-# V 2.1
+# V 2.4.1
 # https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/deploy/installation/
 # helm install aws-load-balancer-controller eks/aws-load-balancer-controller -n kube-system --set clusterName=<cluster-name>
 resource "helm_release" "aws-load-balancer-controller" {
-  depends_on = [kubectl_manifest.crds]
-  name       = "aws-load-balancer-controller"
-  namespace  = "kube-system"
-  repository = "https://aws.github.io/eks-charts"
-  chart      = "aws-load-balancer-controller"
-  version    = "1.1.4" # appVersion: v2.1.2
+  depends_on   = [kubectl_manifest.crds]
+  name         = "aws-load-balancer-controller"
+  namespace    = "kube-system"
+  repository   = "https://aws.github.io/eks-charts"
+  chart        = "aws-load-balancer-controller"
+  version      = "1.4.1" # appVersion: v2.4.1
```

Comment:
Why not let the consumer of this module decide which version of the Helm chart they want to deploy?
I think hardcoding the value and only updating the helm chart with a new module version is too restrictive.

Reply:
Wouldn't that be a leaky abstraction?

Reply:
Thanks for the reply and I see your point, totally agree!

I think my main "complaint" is that we aren't very up to date with the upstream releases...

Reply:
Yep, hence my attempt to get that fixed (for now)